packages/create/templates/hello-express/src/operations.ts

```diff
@@ -0,0 +1,81 @@
+// Welcome to DBOS!
+
+// This is a sample "Hello" app built with DBOS, Express.js, and Knex.
+// It greets visitors and keeps track of how many times each visitor has been greeted.
+
+// First let's import express and DBOS
+import express from "express";
+import { DBOS } from "@dbos-inc/dbos-sdk";
+
+// Then, let's declare a type representing the "dbos_hello" database table
+export interface dbos_hello {
+  name: string;
+  greet_count: number;
+}
+
+// Now let's define a class with some static functions.
+// DBOS uses TypeScript decorators to automatically make your functions reliable, so they need to be static.
+export class Hello {
+  // This function greets a user and increments the greet count in the database.
+  // The @DBOS.transaction() decorator ensures that this function runs as a transaction, only once.
```

Comment:
```suggestion
  // The @DBOS.transaction() decorator ensures that this function runs as a database transaction.
```
Removed "only once" because it sounds confusing.

Reply:
That was a copy+paste, will change both.